.travis.yml

```diff
@@ -5,16 +5,40 @@ services:
   - docker
 
 matrix:
+  # Lets build be marked complete if all required matrix is finished
+  # https://docs.travis-ci.com/user/build-matrix/#fast-finishing
+  fast_finish: true
+
   include:
+  # Docker w/ framework
   - os: linux
-    env: TYPE="conda" CONDA=2.7
+    env: TYPE="docker"
     sudo: required
+
+  # Conda w/o mantid-framework
   - os: linux
-    env: TYPE="conda" CONDA=3.6
+    env: TYPE="conda" CONDA=2.7.14 PKG="mantid-total-scattering-python-wrapper"
```

Comment:
Is it possible to drop the patch level from the version specification like in the 3.6 case?

Reply:
Ah, yes, probably can for the wrapper. We are stuck for the one with the mantid-framework included. Good catch